models/Catalog.js

```diff
@@ -4,19 +4,31 @@ const entity = {
   products: '/products',
   cost: '/products/cost',
   packages: '/packages',
-  basePackages: '/packages/base'
+  basePackages: '/packages/base',
 };
 
-const getProductsCost = options => axiosPost(entity.cost, options);
-const createPackage = options => axiosPost(entity.packages, options);
+// const getProductsCost = options => axiosPost(entity.cost, options);
+const getProductsCost = (array) => {
+  let queryStr = '?';
+
+  array.forEach((element) => {
+    queryStr += 'ids=' + element + '&';
```

Comment:
How many query string do we need? Why are you creating string for every id.  Do you now array.join(',')?